package.json

```diff
@@ -22,7 +22,7 @@
   },
   "dependencies": {
     "react": "^16.3.1",
-    "react-native": "^0.55.2"
+    "react-native": "^0.55.3"
```

Comment:
@tbergq Do we have to upgrade our `package.json` from build folder as well? Maybe no, because there is `^...` anyway? :)